engine/battle_anims/functions.asm

```diff
@@ -3614,7 +3614,7 @@ BattleAnimFunction_MetronomeHand:
 
 BattleAnimFunction_MetronomeSparkleSketch:
 ; Sideways wave motion while also moving downward until it disappears at y coord $20
-; Obj Param: Is used internally only
+; Obj Param: It's used internally only
```

Comment:
I'd remove the verb entirely to match the way other comments are written. (`Obj Param: used internally only`.)

Reply:
On my way.